### DISCUSSION TITLE
global.json rollForward to latestMajor?

### DISCUSSION THREAD

**Discussion by eNeRGy164** [fluentassertions/fluentassertions#1665] [General]
I have a clean system with only Visual Studio 2022 which only comes with .NET SDK 6.0.
On this system I can't build FluentAssertions as it demands SDK 5.0.xxx as required by the global.json file.

If I change the `rollForward` policy to `latestMajor` it works fine using the .NET 6 SDK.

Would it be a problem to change this policy in the repository?

**Comment by jnyrup** [comment]
Fine by me. 
The only risk I can imagine is if we hit a (breaking) change in a newer compiler version.

**Reply by eNeRGy164** [reply]
But in the build pipeline .NET 6 is not installed, so it will use .NET 5 there, it's more of a convenience for local machines.

**Comment by dennisdoomen** [comment]
Find by me too
